src/mesh/mesh_smoother_vsmoother.C

```diff
@@ -145,7 +145,7 @@ void VariationalMeshSmoother::smooth(unsigned int)
       system()->solve();
     }
 
-  libmesh_catch (std::runtime_error& e)
+  libmesh_catch(std::runtime_error& e)
     {
       throw std::runtime_error(std::string("VariationalMeshSmoother: issue encountered during solve:\n") + e.what());
```

Comment:
I've seen the other discussion regarding the `-fno-exceptions` flag not actually working right now, but I wanted to add that it doesn't make sense to use `throw` either if exceptions are disabled... you'll want to change that to a `libmesh_error_msg()` I think.

Reply:
If exceptions are disabled, our `libmesh_catch(foo)` lines turn into `if (0)` ... and I thought that was clever, but clearly that's not sufficient in every case.  We're going to need to `#ifdef LIBMESH_ENABLE_EXCEPTIONS` guard the whole catch block.